src/style.css

```
@@ -1,3 +1,74 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
```

body doesn't have default padding. html doesn't have both
```suggestion
body {
  margin: 0;
}
```